src/memphis.ts

```diff
@@ -186,6 +194,15 @@ class Memphis {
                 this.brokerStats = await this.brokerManager.jetstreamManager();
                 this.isConnectionActive = true;
                 this._configurationsListener();
+                 this.consumeHandlers.forEach(
```

Comment:
please switch to for of and use await syntax instead of .then syntax

Reply:
I've sent a commit with the update